include/xrpl/protocol/detail/transactions.macro

```diff
@@ -465,6 +465,49 @@ TRANSACTION(ttPERMISSIONED_DOMAIN_DELETE, 63, PermissionedDomainDelete, ({
     {sfDomainID, soeREQUIRED},
 }))
 
+/** This transaction creates a single asset vault. */
+TRANSACTION(ttVAULT_CREATE, 64, VaultCreate, ({
+    {sfAsset, soeREQUIRED, soeMPTSupported},
+    {sfAssetMaximum, soeOPTIONAL},
+    {sfMPTokenMetadata, soeOPTIONAL},
+    {sfDomainID, soeOPTIONAL}, // PermissionedDomainID
+    {sfWithdrawalPolicy, soeOPTIONAL},
+    {sfData, soeOPTIONAL},
+}))
+
+/** This transaction updates a single asset vault. */
+TRANSACTION(ttVAULT_SET, 65, VaultSet, ({
+    {sfVaultID, soeREQUIRED},
+    {sfAssetMaximum, soeOPTIONAL},
+    {sfDomainID, soeOPTIONAL}, // PermissionedDomainID
+    {sfData, soeOPTIONAL},
+}))
+
+/** This transaction deletes a single asset vault. */
+TRANSACTION(ttVAULT_DELETE, 66, VaultDelete, ({
+    {sfVaultID, soeREQUIRED},
+}))
+
+/** This transaction trades assets for shares with a vault. */
+TRANSACTION(ttVAULT_DEPOSIT, 67, VaultDeposit, ({
+    {sfVaultID, soeREQUIRED},
+    {sfAmount, soeREQUIRED, soeMPTSupported},
+}))
+
+/** This transaction trades shares for assets with a vault. */
+TRANSACTION(ttVAULT_WITHDRAW, 68, VaultWithdraw, ({
+    {sfVaultID, soeREQUIRED},
+    {sfAmount, soeREQUIRED, soeMPTSupported},
+    {sfDestination, soeOPTIONAL},
+}))
+
+/** This transaction trades shares for assets with a vault. */
```

Comment:
Perhaps this comment needs to be modified

Reply:
thanks, done